src/react/components/common/tagInput/tagInputToolbar.tsx

```diff
@@ -116,14 +125,25 @@ export default class TagInputToolbar extends React.Component<ITagInputToolbarPro
                     );
                 } else if (itemConfig.category === Categories.Separator) {
                     return (<div className="tag-input-toolbar-separator" key={itemConfig.displayName}></div>);
-                } else if (itemConfig.category === Categories.Modifier) {
+                } else if (itemConfig.category === Categories.RenameModifier) {
+                    return (
+                        <IconButton
+                            key={itemConfig.displayName}
+                            disabled={renameModifierDisabled}
+                            title={itemConfig.displayName}
+                            ariaLabel={itemConfig.displayName}
+                            className={renameModifierClassName}
```

Comment:
We could use, instead of all above with class names, ternary and string literals like:
```suggestion
                            className={`tag-input-toolbar-iconbutton-disabled ${renameModifierDisabled ? "tag-input-toolbar-iconbutton-disabled" : " "}`}
```